com/github/mashlol/MySQL.java

```diff
@@ -133,8 +168,11 @@ public void close() {
 		try {
 			con.close();
 		} catch (SQLException e) {
-			System.out.println("A bad call to mysql.close() occurred");	
-			e.printStackTrace();
+                            System.out.println("[StockMarket] " + "General Database error. Could not close SQL connection after use! Enable debug-mode for more info.");
+                            if (StockMarket.debugMode == true) {
+                                e.printStackTrace();
+                            }	
+                            dbstatus = false;
```

Comment:
I was thinking more like:

```
public void logError(String errorMessage, Exception e) {
  System.out.println("[StockMarket] " + errorMessage);
  if (StockMarket.debugMode) {
    e.printStackTrace();
  }
}
```

Not sure if we want to disable the plugin or not, maybe we do.